### PR TITLE
Xccdf openscap

### PR DESCRIPTION
**Pull request by Amndeep7**
Resolves https://github.com/mitre/heimdall_tools/issues/120

### CODE REVIEW

**Comment by Amndeep7** [comment]
@ccoffin pr is here for https://github.com/mitre/heimdall_tools/issues/120

**Comment by Amndeep7** [comment]
1) Need to verify if it's ok to put that sample into our repo for testing purposes
2) Need to verify if it works for @ccoffin's usecase
3) Need to ask @ejaronne some questions about the mapper (see many of the todos, ex. "loaded" as a status)
4) Look into potentially fixing some of the remaining todos (see the rest of the todos, ex. "file"/"counter" as variable names)

File: apps/frontend/src/store/report_intake.ts [hunk 1]
```diff
@@ -251,9 +251,7 @@ export class InspecIntake extends VuexModule {
       if (guessOptions.filename.toLowerCase().endsWith('.nessus')) {
         return 'nessus';
       } else if (
-        guessOptions.data.indexOf(
-          'schemaLocation="http://checklists.nist.gov/xccdf'
-        ) !== -1 ||
+        guessOptions.data.match(/xmlns.*http.*\/xccdf/) ||
```
Comment (georgedias) [inline apps/frontend/src/store/report_intake.ts:254]: Add comments here explaining what is the regex looking for.

File: libs/hdf-converters/src/burpsuite-mapper.ts [hunk 2]
```diff
@@ -64,14 +64,6 @@ function nistTag(input: string): string[] {
...
 }
 
-function parseXml(xml: string): Record<string, unknown> {
-  const options = {
-    attributeNamePrefix: '',
-    textNodeName: 'text',
-    ignoreAttributes: false
-  };
-  return parser.parse(xml, options);
-}
```
Comment (georgedias) [inline libs/hdf-converters/src/burpsuite-mapper.ts:74]: You moved the parseXml? I see that it is imported.

Realized that it was repeated over in all other parsers.

**Review by georgedias — CHANGES_REQUESTED** [review]

**Review by camdenmoors — APPROVED** [review]

**Pull request merged**